src/workerd/server/server.c++

```diff
@@ -3713,26 +3800,26 @@ kj::Own<Server::Service> Server::makeWorker(kj::StringPtr name,
     services.add(kj::mv(globalService));
     services.add(kj::mv(globalService2));
 
-    for (auto& channel: subrequestChannels) {
+    for (auto& channel: def.subrequestChannels) {
       services.add(lookupService(channel.designator, kj::mv(channel.errorContext)));
     }
 
     // Link the ctx.exports self-referential channels. Note that it's important these are added
     // in exactyl the same order as the channels were allocated earlier when we compiled the
     // ctx.exports bindings.
     if (workerService.hasDefaultEntrypoint()) {
-      services.add(KJ_ASSERT_NONNULL(workerService.getEntrypoint(kj::none, kj::none)));
+      services.add(KJ_ASSERT_NONNULL(workerService.getEntrypoint(kj::none, {})));
     }
     for (auto& ep: entrypointNames) {
-      services.add(KJ_ASSERT_NONNULL(workerService.getEntrypoint(ep, kj::none)));
+      services.add(KJ_ASSERT_NONNULL(workerService.getEntrypoint(ep, {})));
```

Comment:
Can you add a comment next to the {} variable for the variable name?

Reply:
Sure: e26c3d1d5

But I wonder if such comments make sense anymore with modern IDEs that can be configured to display the param names automatically.